src/completers/git.zsh

```diff
@@ -342,15 +404,22 @@ _fzf_complete_git-commits() {
     local fzf_options=$1
     shift
 
-    _fzf_complete --ansi --tiebreak=index ${(Q)${(Z+n+)fzf_options}} -- $@$prefix_option < <({
+    _fzf_complete --ansi --tiebreak=index ${(Q)${(Z+n+)fzf_options}} -- $@$prefix_option$prefix_ref < <({
```

Comment:
Note: this case is to handle `--repo=origin:**<TAB>`

Reply:
No, this case is `--force-with-lease=master:**`

Reply:
Alas, I was messing around that then.